cli/internal/env/env.go

```diff
@@ -194,20 +182,14 @@ func (evm EnvironmentVariableMap) fromWildcards(wildcardPatterns []string) (Wild
 		isLiteralLeadingExclamation := strings.HasPrefix(wildcardPattern, "\\!")
 
 		if isExclude {
-			_, excludePattern := wildcardToRegexPattern(wildcardPattern[1:])
+			excludePattern := wildcardToRegexPattern(wildcardPattern[1:])
 			excludePatterns = append(excludePatterns, excludePattern)
 		} else if isLiteralLeadingExclamation {
-			includeLiteral, includePattern := wildcardToRegexPattern(wildcardPattern[1:])
-			if includeLiteral != nil {
-				output.Inclusions[*includeLiteral] = ""
-			}
+			includePattern := wildcardToRegexPattern(wildcardPattern[1:])
 			includePatterns = append(includePatterns, includePattern)
 		} else {
-			includeLiteral, includePattern := wildcardToRegexPattern(wildcardPattern[0:])
+			includePattern := wildcardToRegexPattern(wildcardPattern[0:])
 			includePatterns = append(includePatterns, includePattern)
-			if includeLiteral != nil {
-				output.Inclusions[*includeLiteral] = ""
-			}
```

Comment:
This is how I forced previously-literal env vars to be resolved to `""` _whether specified or not_. That matches the 1.9 behavior.